apollo-normalized-cache-sqlite/gradle.properties

```diff
@@ -1,4 +1,3 @@
 POM_ARTIFACT_ID=apollo-normalized-cache-sqlite
 POM_NAME=Apollo GraphQL SQLite Cache
 POM_DESCRIPTION=Apollo GraphQL SQLite Cache
-POM_PACKAGING=jar
```

Comment:
Same as above, this is required to not override the default `<packaging>klib</packaging>` for the plugin marker